packages/template-block/src/TemplateBlock.spec.ct.tsx

```diff
@@ -0,0 +1,72 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
```

Comment:
Looks for me like there are some tests missing ;)

Reply:
Fixed. More test coverage added.